rubocop-rspec.yml

```diff
@@ -1,29 +1,44 @@
 RSpec:
+  Enabled: true
+
+RSpec/AnyInstance:
   Enabled: false
 
-RSpec/ContextMethod:
-  Enabled: true
+RSpec/DescribeClass:
+  Enabled: false
 
-RSpec/ContextWording:
-  Enabled: true
+RSpec/DescribedClass:
+  Enabled: false
 
-RSpec/EmptyLineAfterExample:
-  Enabled: true
+RSpec/ExampleLength:
+  Enabled: false
 
-RSpec/EmptyLineAfterExampleGroup:
-  Enabled: true
+RSpec/ExpectInHook:
+  Enabled: false
 
-RSpec/EmptyLineAfterFinalLet:
-  Enabled: true
+RSpec/LetSetup:
+  Enabled: false
 
-RSpec/EmptyLineAfterHook:
-  Enabled: true
+RSpec/MessageChain:
+  Enabled: false
 
-RSpec/ExampleWording:
-  Enabled: true
+RSpec/MessageSpies:
+  Enabled: false
 
-RSpec/LetBeforeExamples:
-  Enabled: true
+RSpec/MultipleExpectations:
+  Enabled: false
 
-RSpec/Rails/HttpStatus:
-  Enabled: true
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/NamedSubject:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Enabled: false
+
+RSpec/NotToNot:
```

Comment:
I would like this one to be enabled because when touching another team's project I'm always like "not_to or to_not?" so it'd be nice to have a consensus.... maybe too difficult? just me?

Reply:
updated ef233f7efbd8846ff7704a2bb10c524ef60503a6

Reply:
I remember asking Team 10 devs about that and the opinions were split. Some ppl preferred `to_not`, some `not_to`, and some did not care which one is used. I think it's not a bad idea to allow both.

Reply:
org-wide it might be difficult to reach a consensus but with a given team I think there must be an agreement. Otherwise the specs will read inconsistent. Imagine `not_to` used in a file and suddenly `to_not` being added later by a new team member tasked to add a new feature.

Reply:
> Imagine `not_to` used in a file and suddenly `to_not` being added later by a new team member tasked to add a new feature.

Yes, that's exactly what we have in Tenjin now. And everyone seems to be okay with it. Have we grown wiser? Just got older? I don't know. But AFAICT it's a non-problem.

Reply:
You could apply that reasoning to any style rule.

Reply:
Not true at all 😂 

Most of the style rules we use in Tenjin were picked because people had strong to very strong opinions about them and they wanted the code to be _their way_.
For some reason, this particular rule was the most undisputed of them all. Sure, some people prefer it as `to_not` and others as `not_to`, but - in the end - we were all like "it can be the other way, too" or "I don't really care what we pick here". So both variants are allowed.

Reply:
If there are no hard opinions against either, even more reason to pick a winner and keep moving with life

Reply:
+1